frontend/src/pages/MetricApplication/index.tsx

```diff
@@ -1,76 +1,7 @@
-import { Typography } from 'antd';
-import Spinner from 'components/Spinner';
 import MetricsApplicationContainer from 'container/MetricsApplication';
-import useResourceAttribute from 'hooks/useResourceAttribute';
-import { convertRawQueriesToTraceSelectedTags } from 'hooks/useResourceAttribute/utils';
-import { useEffect, useMemo } from 'react';
-import { connect, useSelector } from 'react-redux';
-import { useParams } from 'react-router-dom';
-import { bindActionCreators } from 'redux';
-import { ThunkDispatch } from 'redux-thunk';
-import {
-	GetInitialData,
-	GetInitialDataProps,
-} from 'store/actions/metrics/getInitialData';
-import { AppState } from 'store/reducers';
-import AppActions from 'types/actions';
-import { GlobalReducer } from 'types/reducer/globalTime';
-import MetricReducer from 'types/reducer/metrics';
-import { Tags } from 'types/reducer/trace';
-
-function MetricsApplication({ getInitialData }: MetricsProps): JSX.Element {
-	const { minTime, maxTime } = useSelector<AppState, GlobalReducer>(
-		(state) => state.globalTime,
-	);
-	const { error, errorMessage, metricsApplicationLoading } = useSelector<
-		AppState,
-		MetricReducer
-	>((state) => state.metrics);
-
-	const { servicename } = useParams<ServiceProps>();
-	const { queries } = useResourceAttribute();
-
-	const selectedTags = useMemo(
-		() => (convertRawQueriesToTraceSelectedTags(queries) as Tags[]) || [],
-		[queries],
-	);
-
-	useEffect(() => {
-		if (servicename !== undefined) {
-			getInitialData({
-				serviceName: servicename,
-				maxTime,
-				minTime,
-				selectedTags,
-			});
-		}
-	}, [servicename, getInitialData, maxTime, minTime, selectedTags]);
-
-	if (metricsApplicationLoading) {
-		return <Spinner tip="Loading..." />;
-	}
-
-	if (error) {
-		return <Typography>{errorMessage}</Typography>;
-	}
 
+function MetricsApplication(): JSX.Element {
 	return <MetricsApplicationContainer />;
```

Comment:
if we are returning only one single component’s no sense writing here

Reply:
As of now, we don't have any logic here, if we have to move we have to move some container login which I think will not be a great idea to do.  What I think is let's keep the pages and it's logic in `/pages/MetricApplication/index.tsx` part in a container and it's logic in `container/MetricApplication`

Thought?

Reply:
Move the logic from container layer to pages layer